src/test/ui/derives/deriving-meta-unknown-trait.stderr

```diff
@@ -3,6 +3,11 @@ error: cannot find derive macro `Eqr` in this scope
    |
 LL | #[derive(Eqr)]
    |          ^^^ help: a derive macro with a similar name exists: `Eq`
+   | 
+  ::: $SRC_DIR/libcore/cmp.rs:LL:COL
+   |
+LL | pub macro Eq($item:item) { /* compiler built-in */ }
+   | ---------------------------------------------------- similarly named derive macro `Eq` defined here
```

Comment:
It seems to me that for some proc and derive macros the new output might be more confusing than helpful for new comers ("how do I do _that_"), but at the same time they'd be nice to have as well for everyone else.